solutions/bronze/cf-1808B.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: cf-1808B
+source: CF 1808B
+title: Playing in a Casino
+author: Amit Singh Bisht
+---
+
+Video Solution: [Official Analysis](https://www.youtube.com/watch?v=FCdL47-AyZs&t=4s&ab_channel=DCCNITA)
```

Comment:
u sure this is official?

Reply:
oh I thought it just had to be like a solution which is why I removed the "official" from "official video solution". should I just remove the video URL?

Reply:
1. use an embed
2. just title it "Video Solution"

Reply:
in my opinion, if the video doesn't help explain your solution better, i probably wouldn't include it.

Reply:
um so I guess i'll just remove it then